extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -69,6 +69,15 @@
     @ConfigItem(defaultValue = "true")
     public boolean transformUnproxyableClasses;
 
+    /**
+     * If set to true, the build will fail if an annotation that would result in an interceptor being created (such as
+     * {@code @Transactional})
+     * is placed on a private method.
+     * If set to false, Quarkus simply logs a warning that the annotation will be ignored.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean failOnPrivateInterceptors;
```

Comment:
It's just a nitpicking but we don't fail _"if a private interceptor is found"_ but if a _"private method is intercepted"_. So maybe something like `failOnInterceptedPrivateMethod`?

Reply:
And the variables in other classes should be renamed as well.

Reply:
@manovotn WDYT?

Reply:
I used your suggestion

Reply:
+1, thanks for changing it